.gitignore

```diff
@@ -2,3 +2,4 @@
 tmp
 .bundle
 vendor
+**/*.gem
```

Comment:
It's an artifact as a result of running `rake` (and not something like `rake build`) - so as an artifact created by a default rake run, it should be ignored or not generated in the first place (or it's confusing - because either you care about the version and should lock the version, or ignore the file, since it's being created every time).

```
g log --name-only -- spec/fixtures/minitest4/vendor/cache/parallel-1.4.0.gem                             (fix_402_alternative_optional_arguments⚡)
commit 47ca16c51e9f10516fa4ebfd7dd166c07cce92cd
Author: grosser <michael@grosser.it>
Date:   Sun Mar 1 07:44:26 2015 -0800
    do not always overwrite
spec/fixtures/minitest4/vendor/cache/parallel-1.4.0.gem
```